integration_tests/cosmoscli.py

```diff
@@ -1315,7 +1315,7 @@ def ica_query_account(self, connid, owner, **kwargs):
             )
         )
 
-    def ica_generate_packet_data(self, tx, memo=None, **kwargs):
+    def ica_generate_packet_data(self, tx, memo=None, encoding="proto3", **kwargs):
```

Comment:
shall we default to proto3json in the precompile, will that be more user friendly?

Reply:
json encoding need support from counterparty

Reply:
seems `proto3json` doesn't work as expected

Reply:
need support from chain-main part I think